gafaelfawr/templates/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "helpers.fullname" . }}
+  labels:
+    app: {{ template "helpers.fullname" . }}
+{{ include "helpers.labels" . | indent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: {{ template "helpers.fullname" . }}
+  template:
+    metadata:
+      labels:
+        name: {{ template "helpers.fullname" . }}
+{{ include "helpers.labels" . | indent 8 }}
+    spec:
+      containers:
+        - name: jwt-authorizer
+          image: "{{ .Values.image | default "lsstdm/jwt_authorizer:0.3.0" }}"
```

Comment:
Probably better to just stick the default in the values.yaml?

Reply:
Oh, good point. Will do.